packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -974,10 +981,28 @@ export async function submitMember(node: api.IZoweTreeNode) {
             { title: "Submit" }
         );
 
-        if (selection == null || selection?.title !== "Submit") {
-            return;
+        return selection != null && selection?.title === "Submit";
+    };
+
+    if (confirmationOption !== "Disabled") {
+        const ownsJob = datasetName.split(".")[0] === nodeProfile.profile?.user?.toUpperCase();
+        if (confirmationOption === "All jobs") {
+            if (!(await showConfirmationDialog())) {
+                return;
+            }
+        } else {
+            if (!ownsJob && confirmationOption === "Other user jobs") {
+                if (!(await showConfirmationDialog())) {
+                    return;
+                }
+            } else if (ownsJob && confirmationOption === "Your jobs") {
```

Comment:
Not suggesting to remove this, but I'm wondering why someone will want only confirmation dialogs for their own jobs and not other user's jobs. 🤔

Reply:
One situation I considered is if a user is working on a critical mainframe job that is not ready to be submitted yet. In that case the user might want to protect themselves from an accidental submission of their own job.